Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -107,205 +107,98 @@ public class InstallationManager : IInstallationManager
             _zipClient = zipClient;
         }
 
-        /// <summary>
-        /// Gets all available packages.
-        /// </summary>
-        /// <returns>Task{List{PackageInfo}}.</returns>
-        public async Task<List<PackageInfo>> GetAvailablePackages(
-            CancellationToken cancellationToken,
-            bool withRegistration = true,
-            string packageType = null,
-            Version applicationVersion = null)
-        {
-            var packages = await GetAvailablePackagesWithoutRegistrationInfo(cancellationToken).ConfigureAwait(false);
-            return FilterPackages(packages, packageType, applicationVersion);
-        }
-
-        /// <summary>
-        /// Gets all available packages.
-        /// </summary>
-        /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>Task{List{PackageInfo}}.</returns>
-        public async Task<List<PackageInfo>> GetAvailablePackagesWithoutRegistrationInfo(CancellationToken cancellationToken)
+        /// <inheritdoc />
+        public async Task<IReadOnlyList<PackageInfo>> GetAvailablePackages(CancellationToken cancellationToken = default)
         {
             using (var response = await _httpClient.SendAsync(
                 new HttpRequestOptions
                 {
                     Url = "https://repo.jellyfin.org/releases/plugin/manifest.json",
                     CancellationToken = cancellationToken,
                     CacheMode = CacheMode.Unconditional,
-                    CacheLength = GetCacheLength()
+                    CacheLength = TimeSpan.FromMinutes(3)
                 },
                 HttpMethod.Get).ConfigureAwait(false))
             using (Stream stream = response.Content)
             {
-                return FilterPackages(await _jsonSerializer.DeserializeFromStreamAsync<PackageInfo[]>(stream).ConfigureAwait(false));
```

Comment:
why no longer filtering packages?

Reply:
are we certain we won't have such packages in the future?

Reply:
I think I disagree. Relying on user input is a bad idea, and this is a user input in a sense.

BTW it would take us both less time to just sort them by version than to continue this discussion anyway xD